core/src/Kokkos_UniqueToken.hpp

```diff
@@ -59,8 +59,8 @@ enum class UniqueTokenScope : int { Instance, Global };
 ///
 /// This object should behave like a ref-counted object, so that when the last
 /// instance is destroy resources are free if needed
-template <typename ExecutionSpace,
-          UniqueTokenScope = UniqueTokenScope::Instance>
+template <typename ExecutionSpace = Kokkos::DefaultExecutionSpace,
```

Comment:
Why did you change?  This is not in the description of the PR.

Reply:
It is now ...

Everything else in Kokkos defaults the execution space. I think this should too.